build.gradle.kts

```diff
@@ -26,32 +26,26 @@ dependencies {
   implementation("org.springframework:spring-jms")
   implementation("com.amazonaws:amazon-sqs-java-messaging-lib:1.0.8")
 
-  implementation("javax.annotation:javax.annotation-api:1.3.2")
-  implementation("javax.xml.bind:jaxb-api:2.3.1")
-  implementation("com.sun.xml.bind:jaxb-impl:2.3.3")
-  implementation("com.sun.xml.bind:jaxb-core:2.3.0.1")
```

Comment:
When I upgraded to v3 nothing broke because the version were being downgraded back to V2 by other dependencies - not quite sure why.

But ultimately if any dependency actually needs these jars then it should depend on them and bring them in as a transitive dependency. None did and things still work, so I don't think they are needed.